keras_hub/src/models/basnet/basnet_presets.py

```diff
@@ -0,0 +1,22 @@
+"""BASNet model preset configurations."""
+
+basnet_presets = {
+    "basnet_resnet18": {
+        "metadata": {
+            "description": "BASNet with a ResNet18 v1 backbone.",
+            "params": 98780872,
+            "official_name": "BASNet",
+            "path": "basnet",
+        },
+        "kaggle_handle": "",  # TODO
```

Comment:
either let's add these before we merge, or remove this entirely before our presets our ready

Reply:
Ok, i will remove this , later will add once presets are ready.